llm/server.go

```diff
@@ -385,8 +385,9 @@ func NewLlamaServer(gpus gpu.GpuInfoList, model string, ggml *GGML, adapters, pr
 			filteredEnv := []string{}
 			for _, ev := range s.cmd.Env {
 				if strings.HasPrefix(ev, "CUDA_") ||
-					strings.HasPrefix(ev, "ROCM_") ||
+					strings.HasPrefix(ev, "ROC") ||
```

Comment:
This is ok – however are there two suffixes we could use here (in case the user has an env var like `ROCK_STORE_SECRET=xxx`